cmd/collectors/restperf/restperf.go

```diff
@@ -24,6 +25,15 @@ const (
 	BILLION       = 1_000_000_000
 )
 
+var qosQuery = "api/cluster/counter/tables/qos"
+var qosVolumeQuery = "api/cluster/counter/tables/qos_volume"
+var qosDetailQuery = "api/cluster/counter/tables/qos_detail"
+var qosDetailVolumeQuery = "api/cluster/counter/tables/qos_detail_volume"
+var qosWorkloadQuery = "api/storage/qos/workloads"
+
+var qosQueries = []string{qosQuery, qosVolumeQuery}
```

Comment:
i think this is used for a contains check - if so, maybe it would be better to replace both of these lines with a map instead of slice?